lib/generators/flatirons/install/install_generator.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Flatirons
+  module Generators
+    class InstallGenerator < ::Rails::Generators::Base
+      source_root File.expand_path('templates', __dir__)
+
+      desc 'Flatirons saas install.'
```

Comment:
we may need a better description,  sets up routing, controllers, run migrations, etc

Reply:
done.